packages/core/src/update/reducer.js

```diff
@@ -15,12 +15,13 @@ import readReducer from '../read/reducer';
 /**
  * Main application reducer.
  *
+ * @param config A configuration object passed from the Engine component
  * @param cursor The cursor representing the state.
  * @param action The action.
  * @returns {*} The new state represented by updated cursor.
  */
-export default function(cursor, action) {
-  // console.log('Reducer: action:', action);
+export default function(config, cursor, action) {
```

Comment:
Thanks for explaining / reminding :)

About the public-api, atm it is there (some moron has put it in): https://github.com/netceteragroup/girders-elements/blob/feature/transformers/packages/core/src/update/index.js

Maybe we just remove it?